inst/examples/person_information/R/extract.R

```diff
@@ -3,14 +3,13 @@
 #' This step should pull data from sources -  local files or database. And load
 #' it into memory ready for transform stage.
 #'
-#' @param path Path to the import project root used for finding any local data.
 #' @param con The active DBI connection for extracting any data.
 #'
 #' @return A list of data frames representing the extracted data.
 #'
 #' @keywords internal
-extract <- function(path, con) {
+extract <- function(con) {
   raw_data <- list()
-  raw_data$people <- read.csv(file.path(path, "data/people.csv"), stringsAsFactors = FALSE)
+  raw_data$people <- read.csv(file.path("data", "people.csv"), stringsAsFactors = FALSE)
```

Comment:
This should just be

```
read.csv("data/people.csv", stringsAsFactors = FALSE)
```

as I was initially confused as to why `file.path` was needed here - and through other affected lines in this PR incl. the vignette

Reply:
This might be some confusion on my end. I thought we still needed `file.path` here to fix file separator for a particular platform. Windows uses `\`? So hard coding the path as `data/people.csv` wouldn't work for windows users? Is that true or does R work around that some other way?

Reply:
Both `a\\b` and `a/b` work on windows from R (as you can see in the original code that used `data/people.csv` as the second arg to `file.path`.  In fact, the help for `?file.path` says

> The components are by default separated by ‘/’ (not ‘\’) on
> Windows.

We should probably put some appveyor tests on this package in any case at some point (not in this PR!) though